src/api/SearchFacetBucket.js

```diff
@@ -78,7 +78,7 @@ export default class SearchFacetBucket {
       if (this.max) {
         result = `${this.min} - ${this.max}`;
       }
-      result = this.min;
+      result = `${this.min}`;
```

Comment:
👍 The return type in the signature of the function is "string" so this should probably already have been like this.